libraries/AP_ExternalAHRS/AP_ExternalAHRS_VectorNav.cpp

```diff
@@ -286,28 +286,29 @@ bool AP_ExternalAHRS_VectorNav::check_uart()
     return true;
 }
 
-// Send command to read given register number and wait for response
+// Send command and wait for response
 // Only run from thread! This blocks until a response is received
 #define READ_REQUEST_RETRY_MS 500
-void AP_ExternalAHRS_VectorNav::wait_register_responce(const uint8_t register_num)
+void AP_ExternalAHRS_VectorNav::send_command_blocking()
```

Comment:
You're not blocking on sending, you're blocking on the processing of the message.


```suggestion
void AP_ExternalAHRS_VectorNav::process_command()
```

Reply:
I am sending the command, and that's the primary action of the method. The blocking on response is secondary functionality to assure that the command I sent was accepted. To me, "process_command" implies I receive a command and I have to manipulate it in some way.

Reply:
My objection is that "blocking" when used with "send" is usually waiting for a message to be sent, *not* a response to be received, thus searching for a better term.

I do use `run_command` elsewhere, that might be better?

Reply:
@peterbarker 

I've pushed to address this comment, renaming "send_command_blocking" to "run_command"